docs/discover/context.asciidoc

```diff
@@ -0,0 +1,44 @@
+[[document-context]]
+== Viewing Document Context
+
+For certain applications it can be useful to inspect a window of documents 
+surrounding a specific event. The context view enables you to do just that for
+index patterns that are configured to contain time-based events.
+
+To show the context surrounding an anchor document, click the *Expand* button 
+image:images/ExpandButton.jpg[Expand Button] to the left of the document's 
+table entry and then click the *View surrounding documents* link.
+
+image::images/Expanded-Document.png[Expanded Document]
+
```

Comment:
This is a very minor formatting issue, but I've been told that adding `{nbsp}` and then a new line is a hack that will correctly space between the image and the text. Right now when you build the docs, they are very close together.